src/main/java/org/trigon/hardware/rev/spark/SparkSignalThread.java

```diff
@@ -47,11 +46,18 @@ private SparkSignalThread() {
         if (ACTIVE) {
             Notifier notifier = new Notifier(this::periodic);
             notifier.setName("SparkSignalThread");
-            notifier.startPeriodic(1.0 / super.odometryFrequencyHertz);
+            notifier.startPeriodic(1.0 / super.threadFrequencyHertz);
         }
     }
 
-    public Queue<Double> registerSignal(DoubleSupplier signal) {
+    /**
+     * Registers a threaded signal to be read asynchronously.
+     * Threaded signals use threading to process certain signals separately. Should be used for signals that need to be updated at a high frequency such as odometry.
```

Comment:
Threaded signals use threading to process certain signals separately at a faster rate.
Keep the jdoc consistent across all threaded stuff